### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,23 +3,12 @@
...
     <h1 align="center">Yii Framework Twitter Bootstrap 5 Extension</h1>
     <br>
 </p>
 
-This [Yii Framework] extension encapsulates [Twitter Bootstrap 5] components
-and plugins in terms of Yii widgets, and thus makes using Bootstrap components/plugins
-in Yii applications extremely easy.
-
-[Yii Framework]:        https://www.yiiframework.com/
-[Twitter Bootstrap 5.2]:  https://getbootstrap.com/docs/5.2/getting-started/introduction/
```
Comment (vjik) [inline README.md:17]: This links are used in text.

File: README.md [hunk 2]
```diff
@@ -3,23 +3,12 @@
         <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px">
     </a>
     <a href="https://getbootstrap.com/" target="_blank" rel="external">
-        <img src="https://v4-alpha.getbootstrap.com/assets/brand/bootstrap-solid.svg" height="100px">
+        <img src="https://v4-alpha.getbootstrap.com/assets/brand/bootstrap-solid.svg" height="80px">
```
Comment (vjik) [inline README.md:6]:
```suggestion
        <img src="https://v4-alpha.getbootstrap.com/assets/brand/bootstrap-solid.svg" height="100px">
```

**Review by vjik — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (luizcmarin) [reply README.md:6]: I kept all Yii logos at 100px to give more emphasis to the second logo, at 80px.
Yii is the star of the show, not a supporting character.

Comment (vjik) [reply README.md]: It looks better when sizes are equal.

File: docs/guide/pt-BR/installation.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ A maneira recomendada para instalar esta extensão é através do [composer](htt
 Então rode
 
 ```
-php composer.phar require --prefer-dist yiisoft/yii-bootstrap4
+composer require yiisoft/yii-bootstrap4
```
Comment (vjik) [inline docs/guide/pt-BR/installation.md:11]:
```suggestion
composer require yiisoft/yii-bootstrap5
```

File: docs/guide/ru/installation.md [hunk 4]
```diff
@@ -8,7 +8,7 @@
 Для этого запустите команду
 
 ```
-php composer.phar require --prefer-dist yiisoft/yii-bootstrap4
+composer require yiisoft/yii-bootstrap4
```
Comment (vjik) [inline docs/guide/ru/installation.md:11]:
```suggestion
composer require yiisoft/yii-bootstrap5
```

File: docs/guide/ukr/installation.md [hunk 5]
```diff
@@ -8,7 +8,7 @@
 Запустіть
 
 ```
-php composer.phar require --prefer-dist yiisoft/yii-bootstrap4
+composer require yiisoft/yii-bootstrap4
```
Comment (vjik) [inline docs/guide/ukr/installation.md:11]:
```suggestion
composer require yiisoft/yii-bootstrap5
```

File: docs/guide/zh-CN/installation.md [hunk 6]
```diff
@@ -8,7 +8,7 @@
 直接运行
 
 ```
-php composer.phar require --prefer-dist yiisoft/yii-bootstrap4
+composer require yiisoft/yii-bootstrap4
```
Comment (vjik) [inline docs/guide/zh-CN/installation.md:11]:
```suggestion
composer require yiisoft/yii-bootstrap5
```

File: docs/internals.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+```shell
+./vendor/bin/rector
+```
+
+## Dependencies
+
+This package uses [composer-require-checker](https://github.com/maglnet/ComposerRequireChecker) to check if all dependencies are correctly defined in `composer.json`.
+
+To run the checker, execute the following command:
```
Comment (vjik) [inline docs/internals.md:41]:
```suggestion
This package uses [composer-require-checker](https://github.com/maglnet/ComposerRequireChecker) to check if
all dependencies are correctly defined in `composer.json`. To run the checker, execute the following command:
```

File: README.md [hunk 8]
```diff
@@ -1,25 +1,14 @@
 <p align="center">
     <a href="https://github.com/yiisoft" target="_blank">
-        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px">
+        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px" alt="Yii">
     </a>
-    <a href="https://getbootstrap.com/" target="_blank" rel="external">
-        <img src="https://v4-alpha.getbootstrap.com/assets/brand/bootstrap-solid.svg" height="100px">
+    <a href="https://getbootstrap.com/" target="_blank">
+        <img src="https://v4-alpha.getbootstrap.com/assets/brand/bootstrap-solid.svg" height="80px" alt="Bootstrap">
```
Comment (vjik) [inline README.md:6]:
```suggestion
        <img src="https://v4-alpha.getbootstrap.com/assets/brand/bootstrap-solid.svg" height="100px" alt="Bootstrap">
```

**Review by vjik — APPROVED** [review]

**Pull request merged**